README.md

```diff
@@ -49,9 +54,18 @@ e.g:
 <image x="14%" y="20%" />
 ```
 
-### Featured Badges
+## FeaturedBadge
 
-If the description you pass in matches one of the featured badges available, the featured badge will instead be returned and any customisation will be lost.
+If the description you pass in matches one of the featured badges available, the featured badge will instead be returned and any customization will be lost.
```

Comment:
I think this is actually wrong. I think this should just describe the single `name` prop and its options.